app/views/fact_values/index.html.erb

```diff
@@ -9,7 +9,7 @@
       <th><%= sort :name, :as => _("Name")  %></th>
       <th><%= sort :value, :as => s_("FactValue|Value") %></th>
       <th><%= _('Reported at') %></th>
-      <th></th>
+      <th><%= _('Distribution chart') %></th>
```

Comment:
the distribution chart header is very long and not trivial imho, i wonder if putting the btn next to the fact name is simple enough (with hover if needed?)